Automattic/pocket-casts-android
# Referrals: profile header

## Description
This adds a referrals profile banner.

## Testing Instructions
1. Launch the app
2. Go to Profiel tab
3. ✅  Notice that Referrals claim pass banner is displayed
4. Disable referrals FF from Profile -> Settings -> Beta features
5. REstart the app
6. ✅ Notice that Referrals claim pass banner is not displayed

## Screenshots or Screencast 

<img width=200 src="https://github.com/user-attachments/assets/0ff18811-5edb-4ace-a1fb-a3905ad28315"/>

<img width=400 src="https://github.com/user-attachments/assets/26581d77-f621-4af9-8e06-fab05aba2355"/>

<img width=500 src="https://github.com/user-attachments/assets/dd4e972d-94a5-4d3b-8a5e-d24ffdfa8530"/>

## Checklist
 
#### I have tested any UI changes...
